scripts/e2e/compare_metrics.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2024 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import difflib
+import sys
+from pathlib import Path
+
+def read_metric_file(file_path):
+    try:
+        with open(file_path, 'r') as f:
+            return f.readlines()
+    except FileNotFoundError:
```

Comment:
I don't think you need to be catching exceptions, they will just bubble up and kill the process anyway